### DISCUSSION TITLE
Slimproto / Squeezebox automation in home assistant

### DISCUSSION THREAD

**Discussion by kobelka** [music-assistant/support#2464] [Q&A] [answered]
Hi,
I can't create an automation in home assistant that makes my squeezebox play something. (But the box works perfectly fine in music assistant)
Does anyone have an example? Or is it not possible at all?
Regards 
Jörg

**Comment by OzGav** [comment]
Have you looked at the examples in the docs?

**Comment by kobelka** [comment]
![image](https://github.com/music-assistant/hass-music-assistant/assets/10710041/cdf8d6fb-cea6-4a9a-b1c7-d695ee78a52c)

I have no service like this... How can I enable it ?

**Reply by OzGav** [reply]
Have you added the HA integration?

**Comment by kobelka** [comment]
![image](https://github.com/music-assistant/hass-music-assistant/assets/10710041/d5351928-af80-4796-ba09-b43bbfebafaf)

Yes, my Home Assistant runs on HAOS

**Reply by OzGav** [reply, marked as answer]
That is the ADDON. You need the INTEGRATION. See the docs.

**Comment by kobelka** [comment]
Thank you, that was very stupid of me. I installed it once and have no idea why, I must have uninstalled it during some tests. Thank you for your support
